### PR TITLE
🌐 [i18n-KO] Translated `gguf.md` to Korean

### PR DESCRIPTION
**Pull request by yijun-lee**
# What does this PR do?

Translated the `gguf.md` file of the documentation to Korean.
Thank you in advance for your review.

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

May you please review this PR?
@jungnerd, @cjfghk5697, @yijun-lee, @mreraser

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/_toctree.yml [hunk 1]
```diff
@@ -135,8 +135,8 @@
     title: 커뮤니티 리소스
   - local: troubleshooting
     title: 문제 해결
-  - local: in_translation
-    title: (번역중) Interoperability with GGUF files
+  - local: gguf
+    title: (번역중) GGUF 파일들과의 상호 운용성
```
Comment (mreraser) [inline docs/source/ko/_toctree.yml:139]:
```suggestion
    title: GGUF 파일들과의 상호 운용성
```
`(번역중)`은 번역 완료 이후에는 삭제해도 될 것 같습니다!

File: docs/source/ko/gguf.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# GGUF와 Transformers의 상호작용 [[gguf-and-interaction-with-transformers]]
+
+GGUF 파일 형식은 [GGML](https://github.com/ggerganov/ggml)과 그에 의존하는 다른 라이브러리, 예를 들어 매우 인기 있는 [llama.cpp](https://github.com/ggerganov/llama.cpp)나 [whisper.cpp](https://github.com/ggerganov/whisper.cpp)에서 모델 추론을 위해 사용됩니다.
```
Comment (mreraser) [inline docs/source/ko/gguf.md:19]:
```suggestion
GGUF 파일 형식은 [GGML](https://github.com/ggerganov/ggml)과 그에 의존하는 다른 라이브러리, 예를 들어 매우 인기 있는 [llama.cpp](https://github.com/ggerganov/llama.cpp)이나 [whisper.cpp](https://github.com/ggerganov/whisper.cpp)에서 추론을 위한 모델을 저장하는데 사용됩니다.
```
원문을 참고하여 조금더 자연스럽게 번역했습니다.

File: docs/source/ko/gguf.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+from transformers import AutoTokenizer, AutoModelForCausalLM
+
+model_id = "TheBloke/TinyLlama-1.1B-Chat-v1.0-GGUF"
+filename = "tinyllama-1.1b-chat-v1.0.Q6_K.gguf"
+
+tokenizer = AutoTokenizer.from_pretrained(model_id, gguf_file=filename)
+model = AutoModelForCausalLM.from_pretrained(model_id, gguf_file=filename)
+```
+
+이제 PyTorch 생태계에서 모델의 역양자화된 전체 버전에 접근할 수 있으며, 다른 여러 도구들과 결합하여 사용할 수 있습니다.
```
Comment (mreraser) [inline docs/source/ko/gguf.md:89]:
```suggestion
이제 PyTorch 생태계에서 모델의 양자화되지 않은 전체 버전에 접근할 수 있으며, 다른 여러 도구들과 결합하여 사용할 수 있습니다.
```
unquantized는 '양자화 되지 않은'으로 보는 것이 더 적절한 것 같아 위와 같이 수정하였습니다.

Comment (mreraser) [reply docs/source/ko/gguf.md:89]: Review complete! 👍

File: docs/source/ko/gguf.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- LLaMa
+- Mistral
+- Qwen2
+- Qwen2Moe
+- Phi3
+- Bloom
+
+## 사용 예시 [[example-usage]]
+
+`transformers`에서 `gguf` 파일을 로드하려면 `from_pretrained` 메서드에 `gguf_file` 인수를 지정해야 합니다. 동일한 파일에서 토크나이저와 모델을 로드하는 방법은 다음과 같습니다:
```
Comment (cjfghk5697) [inline docs/source/ko/gguf.md:77]:
```suggestion
`transformers`에서 `gguf` 파일을 로드하려면 `from_pretrained` 메소드에 `gguf_file` 인수를 지정해야 합니다. 동일한 파일에서 토크나이저와 모델을 로드하는 방법은 다음과 같습니다: 
```
glossary에 맞게 수정했습니다

**Review by stevhliu — APPROVED** [review]
LGTM, thanks for your translation! 🤗

**Pull request merged**